packages/cf-style-provider/src/index.js

```diff
@@ -36,11 +38,22 @@ export const createRenderer = opts => {
     enhancers.push(monolithic());
   }
 
-  return createFelaRenderer({
+  const renderer = createFelaRenderer({
     plugins,
     enhancers,
     selectorPrefix: [usedOpts.selectorPrefix]
   });
+
+  renderer.renderFont('Cloudflare Icons', [
+    `data:application/x-font-ttf;charset=utf-8;base64,${cloudflareIcons}`
+  ]);
+
+  const spinAnimationName = renderer.renderKeyframe(() => ({
+    from: { transform: 'rotate(0deg)' },
+    to: { transform: 'rotate(359deg)' }
+  }));
+
+  return { renderer, spinAnimationName };
```

Comment:
Yea, there is now: https://github.com/rofrischmann/fela/tree/master/packages/fela-plugin-embedded
It was originally my [proposal](https://github.com/rofrischmann/fela/pull/238) and it just landed.

**Please, wait with this PR until the new version of Fela is released and we'll use embedded plugin.**